### DISCUSSION TITLE
Handle webhooks based on sender

### DISCUSSION THREAD

**Discussion by rbur0425** [spatie/laravel-webhook-client#193] [General]
Currently, I have my application set to create a webhook with a uuid for example local.test/webhooks/355950cd-c7aa-4188-8651-2073d12c01e4.

I have one Handler job that processes the information.However, now my users are requesting they can receive webhooks from another third party with the same url...

What is the best way to process a webhook based on the url/host it is coming from? 

In Process.php we do not have access to the request only the webhookCall.

**Comment by QuentinGab** [comment]
The Processor is calling:
```php
protected function storeWebhook(): WebhookCall
{
    return $this->config->webhookModel::storeWebhook($this->config, $this->request);
}
```
so you should use your own `WebhookCall` and customize the storeWebhook method to store the request or part of it in the DB
